<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDbReduceModelDuplicates</class>
 <widget class="QDialog" name="CDbReduceModelDuplicates">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reduce models</string>
  </property>
  <layout class="QVBoxLayout" name="vl_ReduceModelDuplicate" stretch="1,0">
   <item>
    <widget class="swift::gui::COverlayMessagesFrame" name="fr_Overlay">
     <layout class="QVBoxLayout" name="vl_OverlayFrame" stretch="0,1">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QGroupBox" name="gb_Selection">
        <property name="title">
         <string>Selection</string>
        </property>
        <layout class="QFormLayout" name="fl_Selection">
         <item row="0" column="0">
          <widget class="QLabel" name="lbl_Models">
           <property name="text">
            <string>Models:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="le_Models">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lbl_Distributor">
           <property name="text">
            <string>Preferred distributor:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="le_Distributor"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lnl_Mode">
           <property name="text">
            <string>Reduction mode:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QFrame" name="fr_Mode">
           <layout class="QHBoxLayout" name="hl_Mode">
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item alignment="Qt::AlignLeft">
             <widget class="QRadioButton" name="rb_SameLiveryAndAircraft">
              <property name="text">
               <string>with exactly same livery and aircraft</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rb_SameAirlineAndAircraft">
              <property name="text">
               <string>with same airline and aircraft</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="hs_Mode">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QFrame" name="fr_Button">
           <layout class="QHBoxLayout" name="hl_Buttons">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QProgressBar" name="pb_Progress">
              <property name="value">
               <number>24</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pb_Stop">
              <property name="text">
               <string>stop</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pb_Run">
              <property name="text">
               <string>run</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gb_RemoveModels">
        <property name="title">
         <string>Remove models</string>
        </property>
        <layout class="QVBoxLayout" name="vl_RemoveModels">
         <item>
          <widget class="swift::gui::views::CAircraftModelView" name="tvp_RemoveModels">
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="bb_ReduceModelDuplicates">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>swift::gui::views::CAircraftModelView</class>
   <extends>QTableView</extends>
   <header>gui/views/aircraftmodelview.h</header>
  </customwidget>
  <customwidget>
   <class>swift::gui::COverlayMessagesFrame</class>
   <extends>QFrame</extends>
   <header>gui/overlaymessagesframe.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>le_Models</tabstop>
  <tabstop>le_Distributor</tabstop>
  <tabstop>rb_SameLiveryAndAircraft</tabstop>
  <tabstop>rb_SameAirlineAndAircraft</tabstop>
  <tabstop>pb_Stop</tabstop>
  <tabstop>pb_Run</tabstop>
  <tabstop>tvp_RemoveModels</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>bb_ReduceModelDuplicates</sender>
   <signal>accepted()</signal>
   <receiver>CDbReduceModelDuplicates</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bb_ReduceModelDuplicates</sender>
   <signal>rejected()</signal>
   <receiver>CDbReduceModelDuplicates</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
